code/modules/atmospherics/machinery/pipes/multiz.dm

```diff
@@ -19,9 +19,6 @@
 	var/mutable_appearance/pipe = null
 	var/obj/machinery/atmospherics/front_node = null
 
-// Do not hide when under floor
-/obj/machinery/atmospherics/pipe/multiz/hide(i)
-	return
```

Comment:
do these hide now

Reply:
Apologies, oversight. I have added "hide = FALSE" var to address
![image](https://user-images.githubusercontent.com/62388554/220446377-db6d88a8-e9ea-484a-b443-4bb31ac82b7d.png)